servers/rendering/renderer_rd/renderer_compositor_rd.cpp

```diff
@@ -81,7 +85,11 @@ void RendererCompositorRD::blit_render_targets_to_screen(DisplayServer::WindowID
 		blit.push_constant.k2 = p_render_targets[i].lens_distortion.k2;
 		blit.push_constant.upscale = p_render_targets[i].lens_distortion.upscale;
 		blit.push_constant.aspect_ratio = p_render_targets[i].lens_distortion.aspect_ratio;
-		blit.push_constant.convert_to_srgb = texture_storage->render_target_is_using_hdr(p_render_targets[i].render_target);
+		blit.push_constant.source_is_srgb = !texture_storage->render_target_is_using_hdr(p_render_targets[i].render_target);
+		blit.push_constant.target_color_space = color_space;
+		blit.push_constant.max_display_luminance = max_luminance;
+		blit.push_constant.padding[0] = 0.0;
+		blit.push_constant.padding[1] = 0.0;
```

Comment:
We tend to call the padding variables just `pad` instead of `padding`. I suggest sticking to that convention to make them easier to search.